### PR TITLE
Implement "Panic button"

### PR DESCRIPTION
**Pull request by lorcalhost**

### CODE REVIEW

File: btb_manager_telegram/buttons.py [hunk 1]
```diff
@@ -432,3 +433,103 @@ def update_btb():
...
+
+def panic_btn():
+    logger.info("Panic Button button pressed.")
+
+    # Check if open orders / not in usd
+    db_file_path = f"{settings.ROOT_PATH}data/crypto_trading.db"
+    if not os.path.exists(db_file_path):
+        return ["ERROR: Database file not found\.", -1]
+
+    user_cfg_file_path = f"{settings.ROOT_PATH}user.cfg"
```
Comment (DmytroLitvinov) [inline btb_manager_telegram/buttons.py:446]: We can use here also `os.path.join`

Comment (lorcalhost) [reply btb_manager_telegram/buttons.py:446]: I believe it's an improvement but not completely necessary.  
We are sure that the string `settings.ROOT_PATH` already ends with either / or \ depending on the platform it's being run from.

Comment (DmytroLitvinov) [reply btb_manager_telegram/buttons.py:446]: For sure this is improvements which will prevent of bug which can be when `settings.ROOT_PATH` will be without slash at the end. After getting an error of that - we will use in the `os` module. 
As for me - when we are manipulating with with pathname - it is better to use `os.path` module.

**Comment by halilural** [comment]
@lorcalhost hi, we are waiting impatiently :)

**Comment by lorcalhost** [comment]
Hi @halilural ,
sadly I'm unable to focus only on open source development as it doesn't pay the bills.  
`BTB-manager-telegram` is a passion project I work on in my free time while most of the time I'm either studying or doing some gigs.  
If you would like to support the project development you can do so from [here](https://paypal.me/callegari).  

This feature will be released but I'm unable to give an ETA.

**Comment by AnasAlmomany** [comment]
Can you guys merge this branch please, Thanks for the awesome work

**Comment by halilural** [comment]
> Hi @halilural ,
> sadly I'm unable to focus only on open source development as it doesn't pay the bills.
> `BTB-manager-telegram` is a passion project I work on in my free time, while most of the time I'm either studying or doing some gigs.
> If you would like to support the project development you can do so from [here](https://paypal.me/callegari).
> 
> This feature will be released but I'm unable to give an ETA.

Neverthless, I appreciate your work, thanks a lot, unfortunately the paypal is banned in my country.

**Comment by lorcalhost** [comment]
@AnasAlmomany did you test this branch? Did you have issues with it?

**Comment by AnasAlmomany** [comment]
@lorcalhost  yes it's working fine but there is a conflict that should be fixed to be merged.

**Pull request merged**